build error OS/X Maverics

Can follow up with more info as necessary

    julia> versioninfo()
    Julia Version 0.4.0-dev+6070
    Commit f9048e3 (2015-07-18 20:41 UTC)
    Platform Info:
      System: Darwin (x86_64-apple-darwin13.4.0)
      CPU: Intel(R) Core(TM) i7-4850HQ CPU @ 2.30GHz
      WORD_SIZE: 64
      BLAS: libopenblas (USE64BITINT DYNAMIC_ARCH NO_AFFINITY Haswell)
      LAPACK: libopenblas
      LIBM: libopenlibm
      LLVM: libLLVM-svn

    julia> Pkg.clone("https://github.com/Keno/Cxx.jl.git")
    INFO: Cloning Cxx from https://github.com/Keno/Cxx.jl.git
    INFO: Computing changes...

    julia> Pkg.build("Cxx");
    INFO: Building Cxx
    Tuning for julia installation at: /Users/jeffw/src/julia/llvm-svn/usr/bin
    CC /Users/jeffw/.julia/v0.4/Cxx/deps/build/bootstrap.o
    ../src/bootstrap.cpp:736:57: error: value of type 'clang::Stmt::child_range' (aka 'iterator_range<child_iterator>') is not contextually convertible to 'bool'
        for (clang::Stmt::child_range I = Node->children(); I; ++I)
                                                            ^
    ../src/bootstrap.cpp:736:60: error: cannot increment value of type 'clang::Stmt::child_range' (aka 'iterator_range<child_iterator>')
        for (clang::Stmt::child_range I = Node->children(); I; ++I)
                                                               ^ ~
    ../src/bootstrap.cpp:737:11: error: indirection requires pointer operand ('clang::Stmt::child_range' (aka 'iterator_range<child_iterator>') invalid)
          if (*I)
              ^~
    ../src/bootstrap.cpp:738:15: error: indirection requires pointer operand ('clang::Stmt::child_range' (aka 'iterator_range<child_iterator>') invalid)
            Visit(*I);
                  ^~
    4 errors generated.

--- a/src/bootstrap.cpp
+++ b/src/bootstrap.cpp
@@ -733,7 +733,7 @@ public:
   }
 
   void VisitStmt(clang::Stmt *Node) {
-    for (clang::Stmt::child_range I = Node->children(); I; ++I)
+    for (clang::StmtIterator I = Node->children().begin(); I != Node->children().end(); ++I)
       if (*I)
         Visit(*I);
   }

